web/src/stream_edit.js

```diff
@@ -47,6 +47,9 @@ function setup_subscriptions_stream_hash(sub) {
 }
 
 export function setup_subscriptions_tab_hash(tab_key_value) {
+    if ($(".right").hasClass("show")) {
```

Comment:
This CSS selector seems way too generate -- please use something more specific.

Reply:
Using ``id`` of a div can be more specific.
``if($("#stream_settings").css("display") !== "none" || $("#stream-creation").css("display") !== "none")
        return ;``
Using ``#stream_settings``  id, if a stream is opened in the right panel and ``#stream-creation``  id , if  ``Create stream``  is opened in the right panel.
As in both of the above case we want the ``URL``  to not change.
Please give your suggestion on this.

Reply:
I think `$("#subscriptions_overlay .right").hasClass("show")` would be better.

Reply:
Yeah this would also do the job!
@timabbott suggestions?

Reply:
I think you can go with @sahil839 's suggestion without @timabbott 's feedback for now.

Reply:
Pushed the suggested changes.
PTAL!